dashboard/app/controllers/plc_enrollment_evaluations_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PlcEnrollmentEvaluationsController < ApplicationController
```

Comment:
Please stop sending me these emails because I have no idea what they are for

On Wed, Feb 17, 2016 at 12:18 PM, Laurel <notifications@github.com> wrote:

> In dashboard/app/controllers/plc_enrollment_evaluations_controller.rb
> <https://github.com/code-dot-org/code-dot-org/pull/6813#discussion_r53108423>
> :
>
> > @@ -0,0 +1,22 @@
> > +class PlcEnrollmentEvaluationsController < ApplicationController
>
> why use Plc sometimes and spell out ProfessionalLearningCourses sometimes?
> Would it make sense to namespace all of these controllers?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/6813/files#r53108423>.
>